Makefile

```diff
@@ -0,0 +1,21 @@
+target: virtualenv run tests clean
+
+VENV=mkfike_env
+PYTHON=$(VENV)/bin/python
+
+virtualenv:
+	@python3 -m venv $(VENV)
+	@. $(VENV)/bin/activate
+
+run: virtualenv
```

Comment:
@LazyDevoCat, I wonder is this command useful? I think you can remove it.

Reply:
I suggest to leave this Block, because if in development propouse we will need to install another packeges (flake8, pylint, bandit, etc.) . It will be easely to remove these packeges with virtual environment.

Reply:
[Solved](https://github.com/LazyDevoCat/convert_temp/pull/22/commits/fcb65771b657f8c1a258ec51af382b003bcb791a)